daemon/container.go

```diff
@@ -1137,6 +1138,31 @@ func (container *Container) shouldRestart() bool {
 		(container.hostConfig.RestartPolicy.Name == "on-failure" && container.ExitCode != 0)
 }
 
+func (container *Container) mountVolumes() error {
```

Comment:
Kind of surprised that this functionality isn't already available via some other func.  Go figure.